server/dbScripts/dbScripts.js

```diff
@@ -1,63 +1,82 @@
-});
-db.rssLink.insert({
-  media: db.media.findOne({value: 'BBC'})._id,
-  categories: [db.category.findOne({value: 'Health'})._id],
-  link: 'http://feeds.bbci.co.uk/news/health/rss.xml'
-});
-db.rssLink.insert({
-  media: db.media.findOne({value: 'BBC'})._id,
-  categories: [
-    db.category.findOne({value: 'Entertainment'})._id,
-    db.category.findOne({value: 'Arts'})._id
-  ],
-  link: 'http://feeds.bbci.co.uk/news/entertainment_and_arts/rss.xml'
-});
-db.rssLink.insert({
-  media: db.media.findOne({value: 'CNN'})._id,
-  categories: [db.category.findOne({value: 'World'})._id],
-  link: 'http://rss.cnn.com/rss/edition_world.rss'
-});
-db.rssLink.insert({
-  media: db.media.findOne({value: 'CNN'})._id,
-  categories: [db.category.findOne({value: 'Technology'})._id],
-  link: 'http://rss.cnn.com/rss/edition_technology.rss'
-});
-db.rssLink.insert({
-  media: db.media.findOne({value: 'CNN'})._id,
-  categories: [db.category.findOne({value: 'Sports'})._id],
-  link: '	http://rss.cnn.com/rss/edition_sport.rss'
-});
-db.rssLink.insert({
-  media: db.media.findOne({value: 'CNN'})._id,
-  categories: [db.category.findOne({value: 'Entertainment'})._id],
-  link: 'http://rss.cnn.com/rss/edition_entertainment.rss'
-});
+// db.rssLink.insert({
+//   media: db.media.findOne({value: 'BBC'})._id,
+//   categories: [db.category.findOne({value: 'World'})._id],
+//   link: 'http://feeds.bbci.co.uk/news/world/rss.xml'
+// });
+// db.rssLink.insert({
+//   media: db.media.findOne({value: 'BBC'})._id,
+//   categories: [db.category.findOne({value: 'Technology'})._id],
+//   link: 'http://feeds.bbci.co.uk/news/technology/rss.xml'
+// });
+// db.rssLink.insert({
+//   media: db.media.findOne({value: 'BBC'})._id,
+//   categories: [db.category.findOne({value: 'Health'})._id],
+//   link: 'http://feeds.bbci.co.uk/news/health/rss.xml'
+// });
+// db.rssLink.insert({
+//   media: db.media.findOne({value: 'BBC'})._id,
+//   categories: [
+//     db.category.findOne({value: 'Entertainment'})._id,
+//     db.category.findOne({value: 'Arts'})._id
+//   ],
+//   link: 'http://feeds.bbci.co.uk/news/entertainment_and_arts/rss.xml'
+// });
+// db.rssLink.insert({
+//   media: db.media.findOne({value: 'CNN'})._id,
+//   categories: [db.category.findOne({value: 'World'})._id],
+//   link: 'http://rss.cnn.com/rss/edition_world.rss'
+// });
+// db.rssLink.insert({
+//   media: db.media.findOne({value: 'CNN'})._id,
+//   categories: [db.category.findOne({value: 'Technology'})._id],
+//   link: 'http://rss.cnn.com/rss/edition_technology.rss'
+// });
+// db.rssLink.insert({
+//   media: db.media.findOne({value: 'CNN'})._id,
+//   categories: [db.category.findOne({value: 'Sports'})._id],
+//   link: '	http://rss.cnn.com/rss/edition_sport.rss'
+// });
+// db.rssLink.insert({
+//   media: db.media.findOne({value: 'CNN'})._id,
+//   categories: [db.category.findOne({value: 'Entertainment'})._id],
+//   link: 'http://rss.cnn.com/rss/edition_entertainment.rss'
+// });
```

Comment:
I think we better remove them

Reply:
will remove after finishing the scrape feature is fully built..now the category part is still missing